api/v1/mongodbcommunity_types.go

```diff
@@ -602,6 +639,18 @@ func (m MongoDBCommunity) CurrentReplicas() int {
 	return m.Status.CurrentStatefulSetReplicas
 }
 
+func (m MongoDBCommunity) ForcedIndividualScaling() bool {
```

Comment:
This method always returns false - Is this needed?

Reply:
For `automationConfigReplicasScaler` this method is different. The meaning is: "with Replica set members, you can Scale 0 -> 3 at once (only at the beginning), with Arbiters you never want that, they always scale one-by-one".

The controller can create a Replica Set with 3 members at once, but after they have been created, the arbiters will be added one by one, because after creation, scaling needs to be done per member.

Let me know if that explains the situation

Reply:
sounds good, maybe we can add it as a comment in the function for posterity.